supabase/migrations/20260308121933_restrict_global_stats_access.sql

```
@@ -9,6 +9,12 @@
 -- Remove the permissive policy that allowed anonymous reads.
 DROP POLICY IF EXISTS "Allow anon to select" ON public.global_stats;
 
+CREATE POLICY "Allow none to select"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)